.travis.yml

```diff
@@ -56,9 +56,12 @@ install:
     - echo ${TEST}
     - echo ${PYTHON}
 
-    - if [ "${QT}" == "pyqt" ]; then
+    - if [ "${QT}" == "pyqt5" ]; then
           conda install pyqt --yes;
       fi;
+    - if [ "${QT}" == "pyqt4" ]; then
+          apt-get install python-qt4 python-qt4-gl;
```

Comment:
You should be able to `conda install pyqt<5` which will pull ` pyqt 4.11.4-py35_4`